README.md

```diff
@@ -1,14 +1,45 @@
 # Documentation template for OS2 projects
 
-## 🏗️ Getting started 
+## How to use this template for documenting your OS2 project
 
-This is a *bare-minimum* template to create a [Jekyll][Jekyll] site that:
+## 👷 Maintainers guide
 
-- uses the [Just the Docs][Just the Docs] theme;
-- can be built and published on [GitHub Pages][GitHub Pages];
-- can be built and previewed locally, and published on other platforms.
+*In OS2 projects the maintainer team is the group of individuals assigned to oversee the development, quality, and community engagement of the documentation project.*
 
-[Browse documentation][Just the Docs] to learn more about how to use this theme.
+# Setup
+
+1. Start by [creating a documentation repository in your project, from this template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template), including only the main branch.
```

Comment:
The link should point to the actual template repro. Or am I misreading the sentence "from this template".

Reply:
Sorry I forgot that you the person reading this is actually inside the template and will have the button "use this template" available.